pkg/test/resourcefixture/sets.go

```diff
@@ -54,8 +54,16 @@ func GetBasicTypeSetCover(t *testing.T) []ResourceFixture {
 		return testType == Basic
 	}
 	heavyFilter := func(fixture ResourceFixture) bool {
+
 		// Skip v1alpha1 CRDs when testing set cover as they may not yet be
 		// correctly supported.
+
+		// We do allowlist a few that we are hand-coding / checking.
+		// (Needed to get TestGetBasicTypeSetCover to pass)
```

Comment:
Hmmm, can we make TestGetBasicTypeSetCover() check for v1beta1 services only?

Reply:
Changed `getResourceConfigIdSet` to ignore all v1alpha1 resources.